user_pinger.py

```diff
@@ -208,16 +220,26 @@ def handle_comment(self, comment: praw.models.Comment) -> None:
         except ValueError:
             # no trigger
             return
-        else:
-            self.logger.debug("Ping found in %s", str(comment))
-            try:
-                trigger: str = split[index + 1]
-            except IndexError:
-                self.logger.debug("End of comment with no group specified")
-                return
-            else:
-                self.logger.debug("Found group is %s", trigger)
-                self.handle_ping(trigger, comment)
+        self.logger.debug("Ping found in %s", str(comment))
+        try:
+            token: str = split[index + 1]
+        except IndexError:
+            self.logger.debug("End of comment with no group specified")
+            return
+        self.logger.debug("Comment contained group token %s", token)
+
+        # Strip punctuation like:
+        # !ping WEEBS, what do you folks think of this?
+        trigger: str = token.strip(self.GROUP_STRIP_PUNCT)
+
+        # Validate group name to save having to look up invalid names
+        ret: bool, msg: Optional[str] = self._validate_group_name(trigger)
+        if not ret:
+            self.logger.debug(msg)
+            return
```

Comment:
Would be helpful to let people know why their ping didn't go through

```suggestion
            self._send_error_pm(f"Invalid group name", [f"The group name {trigger} contains invalid characters"], comment.author)
            return
```

Reply:
Whoops, gave that the wrong description